x/prepend.js

```diff
@@ -0,0 +1,54 @@
+const isString = require('./isString');
+
+/**
+ * @name prepend
+ *
+ * @synopsis
+ * ```coffeescript [specscript]
+ * prepend(
+ *   item string|Array,
+ * )(value string|Array) -> string|array
+ * ```
+ *
+ * @description
+ * Prepend a string or an array.
+ *
+ * ```javascript [playground]
+ * import prepend from 'https://unpkg.com/rubico/dist/x/prepend.es.js'
+ *
+ * const myArray = ['orange', 'apple']
+ * const result = prepend(['ananas'])(myArray)
+ * console.log(result) // ['ananas', 'orange', 'apple']
+ *
+ * const myArray = ['orange', 'apple']
+ * const result = prepend('ananas')(myArray)
+ * console.log(result) // ['ananas', 'orange', 'apple']
+ *
+ * const myArray = ['orange', 'apple']
+ * const result = prepend('hello ')('world')
+ * console.log(result) // 'hello world'
+ * ```
+ *
+ * @since 1.7.3
+ */
+
+const prepend = item => function prependFunc(value) {
+
+    if (Array.isArray(value)) {
```

Comment:
let's use the _internal/isArray here, which is just the dereferenced Array.isArray. This is to skip an extra property access step.